README.md

```diff
@@ -19,18 +19,29 @@ https://github.com/darwinai/pl-CT-covidnet
 https://github.com/darwinai/pl-pdfgeneration
 ```
 
-Ensure you build the docker containers for these plugins by:
+2) Download the models into plugin folders:
+    a) Download models COVIDNet-CXR4-B, COVIDNet-SEV-GEO, COVIDNet-SEV-OPC from https://github.com/lindawangg/COVID-Net/blob/master/docs/models.md and place them into the subfolder covidnet/models of pl-covidnet plugin.
+    b) Download model COVIDNet-CT-A from https://drive.google.com/drive/folders/13Cb8yvAW0V_Hh-AvUEDrMEpwLhD3zv-F and place them into the subfolder ct_covidnet/models of pl-CT-covidnet plugin.
+
+3) Build the docker container images for these plugins by running:
+```
+docker build -t local/pl-covidnet .
+docker build -t local/pl-ct-covidnet .
+docker build -t local/pl-pdfgeneration .
 ```
-docker build -t  {name_of_the_plugin} .
 
+4) Type the following command to verify all images were built successfully:
+```
+docker image -ls
 ```
+NOTE: If pl-covidnet plugin fails to build with version compatibility error, open the file requirements.txt and remove version numbers near tensorflow-estimator and tensorboard.
```

Comment:
NOTE:  If the pl-covidnet plugin fails to build ... remove the version numbers near ...